api-spec/protobuf/operator.proto

```diff
@@ -703,3 +708,45 @@ message Page {
   // The size of the page, ie the max length the returning list can have.
   int64 page_size = 2;
 }
+
+message GetMarketReportRequest {
+  // The marker for which returning the report of collected fees.
+  Market market = 1;
+  // time_range fetch market report from time range
+  TimeRange time_range = 2;
+}
+message GetMarketReportsReply {
+  // MarketReport holds info about volume and collected fees
+  MarketReport market_report = 1;
+}
+
+message MarketReport {
+  map<string, int64> total_collected_fees_per_asset = 1;
+  map<string, int64> volume_per_asset = 2;
```

Comment:
maybe better to have 2 dedicated messages instead of generic maps like:
```
message MarketCollectedFees {
   uint64 base_amount = 1;
   uint64 quote_amount = 2;
}
```
and
```
message MarketVolume {
   uint64 base_volume = 1;
   uint64 quote_volume = 2;
}
```
?

Reply:
also, could be worth adding info about the trades? like id, txid, swapRequestMessage and settlement timestamp?

Reply:
@tiero should we add new fields as altafan proposed?

Reply:
But this is the computed amount? so its summing up all trades?

Reply:
> so its summing up all trades? 

"all" trades can be all trades in the last hour, month, year, or in a custom period. I was just wondering if the user might find useful to have some sort of lightweight view on these trades

Reply:
Done